climada/util/coordinates.py

```diff
@@ -801,6 +801,64 @@ def get_region_gridpoints(countries=None, regions=None, resolution=150,
         lat, lon = [ar.ravel() for ar in [lat, lon]]
     return lat, lon
 
+def mapping_point2grid(geometry, ymax, xmin, res):
+    """Given the coordinates of a point, find the index of a grid cell from 
+    a raster into which it falls.
+    
+    Parameters
+    ---------
+    geometry : shapely.geometry.Point object
+        Point which should be evaluated
+    ymax: float
+        coords of top left corner of raster file - y
+    xmin: float
+        coords top left corner of raster file - x
+    res: float or tuple
+        resolution of raster file. Float if res_x=res_y else (res_x, res_y).
+    
+    Returns
+    ------- 
+    col, row : tuple
+        column index and row index in grid matrix where point falls into
+    
+    Raises
+    ------
+    ValueError if Point outside of top left corner of raster
+    """
+    if isinstance(res, tuple):
+        res_x, res_y = (abs(res) for res in res)
+    else:
```

Comment:
A bit more general, would allow not only tuples but also arrays of np.arrays
```suggestion
    if len(res) == 2:
        res_x, res_y = (abs(res) for res in res)
    else:
```

Reply:
a float object has no length, hence len(res) would not work. that's why i do the type checking. i can add tuple _or_ list in the check

Reply:
Aaa yes. Hmmm then the 'pythonic' way would probably be a try...except block ... (better ask for forgiveness than for permission)